deploy/scripts/install.sh

```diff
@@ -43,6 +43,9 @@ if [[ -z "$USER_OBJ_ID" ]]
     echo "To get USER_OBJ_ID, run 'az ad signed-in-user show --query id --output tsv'"
 fi
 
+# Captures the Azure cloud endpoints/suffixes
+az cloud show > $PRJ_ROOT/deploy/cloud_endpoints.json
```

Comment:
Json is currently the default, but it wouldn't hurt to be explicit here.
```suggestion
az cloud show -o json > $PRJ_ROOT/deploy/cloud_endpoints.json
```